pennylane/qnode.py

```diff
@@ -369,6 +371,9 @@ def construct(self, args, kwargs=None):
                     if p.name is None: # ignore keyword arguments
                         self.variable_ops.setdefault(p.idx, []).append((k, idx))
 
+        # generate directed acyclic graph
+        self.cg = CircuitGraph(self.queue, self.ev, self.variable_ops)
```

Comment:
I would opt for either `self.graph` or `self.circuit_graph` to convey the idea that this is a graph -- thoughts?

Reply:
I'm happy with either!

Reply:
I think `circuit` is more informative than `graph` though. Graph describes the underlying data structure, but not what the data structure encapsulates (which is a circuit).

Since `circuit` isn't used for any other data structure/object, we may as well standardize so that `circuit` implies a DAG.

Reply:
Done.